test.js

```diff
@@ -0,0 +1,26 @@
+var expect = require('chai').expect;
+var localeEmoji = require('.');
+
+describe('localeEmoji', function() {
+  var tests = {
+    'de': '🇩🇪',
+    'de-DE': '🇩🇪',
+    'de_DE': '🇩🇪',
+    'de-CH': '🇨🇭',
+    'en': '🇬🇧',
+    'EN': '🇬🇧',
+    'en-GB': '🇬🇧',
+    'en-US': '🇺🇸',
+    'EN-US': '🇺🇸',
+    'EN-us': '🇺🇸',
+    'en-us': '🇺🇸',
+    'sk_Latin_SK': '🇸🇰',
+  };
+
+  Object.keys(tests).forEach(function(from) {
+    var to = tests[from];
+    it('converts "' + from + '" to ' + to, function() {
+      expect(localeEmoji(from)).to.equal(to);
+    });
+  });
+});
```

Comment:
I'm not a huge fan of dynamic tests.

Make any `if` declaration static.  The `expect` inside a `foreach` is 👌 tho.

Also, trailing line break.